docs/resources/service_acl_entries.md

```diff
@@ -199,6 +199,33 @@ resource "fastly_service_acl_entries" "entries" {
 }
 ```
 
+### Reapplying original entries with `managed_entries` if the state of the entries drifts
```

Comment:
@Hiieu might be worth checking with @bengesoff but would the above section "Supporting API and UI ACL updates with ignore_changes" not need to be removed? I don't think using `ignore_changes` would work since Ben's new `manage_*` implementation (or at least I'm not sure if we want to avoid publicly promoting that pattern as it might have unexpected side effects maybe)?

Reply:
Same comment essentially for `docs/resources/service_dictionary_items.md`.

Reply:
As discussed, we've removed `ignore_changes` from the docs and examples, also added a note saying that the built-in meta-argument takes precedence over `manage_x` attributes.